code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PendingIntentUtils.kt

```diff
@@ -106,6 +108,8 @@ internal object PendingIntentUtils {
         intent.setFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP or Intent.FLAG_ACTIVITY_CLEAR_TOP)
         intent.putExtra(PushTemplateConstants.PushPayloadKeys.TAG, tag)
         intent.putExtra(PushTemplateConstants.PushPayloadKeys.STICKY, stickyNotification.toString())
+        // todo revisit if all data is needed for click actions
```

Comment:
good idea. i think we can definitely clean this up in a future pr.

Reply:
`stickyNotification` value is overridden sometimes in the builder class but it can definitely be added to the extas passed in here. Will make the change